Marlin/src/module/ft_motion.h

```diff
@@ -171,16 +172,15 @@ class FTMotion {
 
       typedef struct Shaping {
         uint32_t zi_idx;           // Index of storage in the data point delay vectors.
-        axis_shaping_t x;
-        #if HAS_Y_AXIS
-          axis_shaping_t y;
-        #endif
+
+        TERN0(HAS_X_AXIS, axis_shaping_t x;)
+        TERN0(HAS_Y_AXIS, axis_shaping_t y;)
```

Comment:
Never wrap declarations in macros like `TERN_()` because this obscures them from VSCode IntelliSense and other editors that offer "Go to Declaration…". Use complete `#if`/`#endif` blocks instead. For some enums and data members we have in the past used `OPTITEM` or `OPTCODE` but this is not favored for the cited reason.